codespell_lib/data/dictionary.txt

```diff
@@ -25107,10 +25359,12 @@ warinigs->warnings
 warinings->warnings
 warks->works
 warlking->walking
+warmup->warm up
```

Comment:
Should also be `warm-up`  
This is going to show a lot of false positives even though grammatically that is correct.

Reply:
Added warmup->warm up, warm-up,